eduNEXT/xblock-content-restrictions
# fix: resize xblock iframe on content changes

### Description
Add a simplified script based on [this edx-platform script](https://github.com/openedx/edx-platform/blob/master/lms/templates/courseware/courseware-chromeless.html#L119-L217) in the xblock template to resize the iframe each time we navigate between child components. 

### How to Test
1. Install this branch.
2. Add `content_restrictions` to **Studio** > **Course** > **Settings** > **Advanced Settings**.
3. Add a new component of **Content Restrictions** in a unit.
4. Add some children to the component.
5. Enable and configure password restriction.
6. In the LMS set the correct password, all child components should be rendered correctly.
